carbon-design-system/carbon-addons-iot-react
# fix(time-picker-spinner): don't allow 00 for 12-hour picker

Closes #2893 

**Summary**

- previously the TimePickerSpinner would allow 00:00 as a valid input when using `is12hours` prop. This fixes that and confirms it with tests.

**Change List (commits, features, bugs, etc)**

- properly roll over from 12->1 or 1->12
- added tests to confirm.

**Acceptance Test (how to verify the PR)**

- check TimePickerSpinner story
- turn on `is12hour` prop
- confirm hours roll over from 12->1 or 1->12 in incrementing or decrementing

**Regression Test (how to make sure this PR doesn't break old functionality)**

- confirm it doesn't break roll-over on 24hour time or in minutes.



**Things to look for during review**
